tests/Kernel/TenderRenderTest.php

```diff
@@ -0,0 +1,254 @@
+      'oe_content_departments_field',
+      'oe_content_reference_code_field',
+      'oe_content_tender',
+      'oe_theme_content_tender',
+    ]);
+
+    module_load_include('install', 'oe_theme_content_tender');
+    oe_theme_content_tender_install();
+
+    module_load_include('install', 'oe_content');
+    oe_content_install();
+
+    // Set current user to UID 1, so that by default we can access everything.
+    $account = User::load(1);
+    $this->setCurrentUser($account);
+  }
+
+  /**
+   * Test a call for tender being rendered as a teaser.
+   */
+  public function testTenderTeaser(): void {
+    // Freeze the time at a specific point.
+    $static_time = new DrupalDateTime('2020-02-17 14:00:00', DateTimeItemInterface::STORAGE_TIMEZONE);
+
+    $publication_date = (clone $static_time)->modify('- 5 days');
+    $opening_date = (clone $static_time)->modify('- 3 days');
+    $deadline_date = (clone $static_time)->modify('+ 3 days');
+
+    /** @var \Drupal\Component\Datetime\TimeInterface $datetime */
+    $time = $this->container->get('datetime.time');
+    $time->freezeTime();
+    $time->setTime($static_time->getTimestamp());
+
+    // Create a Call for tender node.
+    /** @var \Drupal\node\Entity\Node $node */
+    $values = [
+      'type' => 'oe_tender',
+      'title' => 'Test Call for tenders node',
+      'oe_publication_date' => [
+        'value' => $publication_date->format('Y-m-d'),
+      ],
+      'oe_tender_opening_date' => [
+        'value' => $opening_date->format('Y-m-d'),
+      ],
+      'oe_tender_deadline' => [
+        'value' => $deadline_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+      ],
+      'oe_reference_code' => 'Call for tenders reference',
+      'oe_departments' => ['http://publications.europa.eu/resource/authority/corporate-body/ABEC', 'http://publications.europa.eu/resource/authority/corporate-body/ACM'],
+      'uid' => 0,
+      'status' => 1,
+    ];
+    $node = Node::create($values);
+    $node->save();
+
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+
+    // Check Open status label and background.
+    $assert = new ListItemAssert();
+    $deadline_date->setTimeZone(new \DateTimeZone('Australia/Sydney'));
+    $expected_values = [
+      'title' => 'Test Call for tenders node',
+      'meta' => 'Call status: Open',
+      'image' => NULL,
+      'description' => new PatternAssertState(new FieldListAssert(), [
+        'items' => [
+          [
+            'label' => 'Reference',
+            'body' => 'Call for tenders reference',
+          ], [
+            'label' => 'Opening date',
+            'body' => $opening_date->format('d F Y'),
+          ], [
+            'label' => 'Deadline date',
+            'body' => $deadline_date->format('d F Y, H:i (T)'),
+          ], [
+            'label' => 'Departments',
+            'body' => "Audit Board of the European Communities\n   |   Arab Common Market",
```

Comment:
Can we move the `   {% if not loop.last %} | {% endif %}` in the template next to the field? so we don't have to assert this  `\n   |   `, or wrap the template in spaceless, whichever works.